### ISSUE TITLE
I dont quite understand how Upsampling works.

### ISSUE THREAD

**Issue by sjgibbs** [AgileStationery/sample-issues#5]
I am trying to implement a fully convolutional neural network which requires an upsampling step before softmaxout. The small score maps need to be upsampled to the same size as the ground-truth label. I tried to use Upsampling layer but it seems the layer requires weights parameters. I simply just want to use bilinear methods to scale the score maps, for example I have 21 score maps with size 64_64. I just need them to be resized to for example 128_128. I dont see the need of those weights. Could anyone explain to me how this Upsampling layer works? If this layer is not what I am looking for, what else can I use to achieve this?

**Comment by sjgibbs** [comment]
From https://github.com/apache/incubator-mxnet/issues/1412
Exhibited lots of clipping in the output
